assets/custom.css

```diff
@@ -432,6 +432,36 @@ input[type="search"]::-webkit-search-results-decoration { display: none; }
     padding: 1rem;
 }
 
+/* HEADER MEGAMENU */
+.header__menu-dropdown {
+    height: 40vw;
```

Comment:
I wouldn't do this. You rarely want to set both width & height. In this case, you should set width like you did but let the height be automatically set by the height of its contents.

On my monitor, this looks HUGE. Just set padding to the container and adjust it that way.

<img width="2559" alt="Screen Shot 2022-03-14 at 9 34 20 PM" src="https://user-images.githubusercontent.com/15237105/158307698-f4481528-cb4c-4b69-90f6-9ab556ab4dfb.png">

Reply:
Changes made. Ready for review

Reply:
looks good!